release-calendar/src/client/components/App.tsx

```diff
@@ -13,10 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import * as React from 'react';
+import {Calendar} from './Calendar';
+import {FAKEEVENTS} from './fakeData';
 import {Header} from './Header';
 
 export const App = (): JSX.Element => {
-  return <Header title='AMP Release Calendar' />;
+  return (
+    <div>
+      <Header title='AMP Release Calendar' />
+      <div className='AMP-Release-Calendar-Full-Calendar'>
```

Comment:
Is this `<div>` necessary?

Reply:
It is because there needs to be parent div pair. I can name it AMP-Release-Calendar though for clarity and eventually for styling

Reply:
Link to React doc about parent div pair? I removed `<div className='AMP-Release-Calendar-Full-Calendar>` and the calendar still worked.

I also don't see `AMP-Release-Calendar-Full-Calendar` defined anywhere in a stylesheet, which gave me the hint that maybe this can be removed. (and a small comment, class names are usually in "kebab case" and easier to work with if they're shorter)